src/runtime/CMakeLists.txt

```diff
@@ -168,20 +169,52 @@ foreach (i IN LISTS RUNTIME_CPP)
     foreach (j IN ITEMS 32 64)
         # -fpic needs special treatment; see below on windows 64bits
         set(fpic -fpic)
-        if (j EQUAL 32)
-            if (i MATCHES "windows_.*")
-                # win32 uses the stdcall calling convention, which is x86-specific
-                set(TARGET "i386-unknown-unknown-unknown")
-            else ()
+        # for the generic windows 64-bit target, we need -fshort-wchar
```

Comment:
I believe the Makefile also need changes corresponding to these

Reply:
There is ~no support for building any Windows targets using the Makefiles, so, probably not

EDIT: oh, wait, for crosscompiling from (say) linux -> windows we do need the Makefile to get this right, so, yes, needs attention